azure/durable_functions/entity.py

```diff
@@ -0,0 +1,114 @@
+from .models import DurableEntityContext
+from .models.entities import OperationResult, EntityState
+from datetime import datetime
+from typing import Callable, Any, List, Dict
+
+
+class Entity:
+    """Durable Entity Class.
+
+    Responsible for execuitng the user-defined entity function.
+    """
+
+    def __init__(self, entity_func: Callable[[DurableEntityContext], None]):
+        """Create a new entity for the user-defined entity.
+
+        Responsible for executing the user-defined entity function
+
+        Parameters
+        ----------
+        entity_func: Callable[[DurableEntityContext], Generator[Any, Any, Any]]
+            The user defined entity function
+        """
+        self.fn: Callable[[DurableEntityContext], None] = entity_func
+
+    def handle(self, context: DurableEntityContext, batch: List[Dict[str, Any]]) -> str:
```

Comment:
Is there a reason the individual operations in a batch are represented as a Dictionary instead of a defined type? It seems we lose a lot of type safety this way.

Reply:
I suppose we could create a proper constructor for it, or at least a type alias. We just did not prioritize it, but I agree with the sentiment. Should we separate this into a separate issue?

Reply:
If the timing is tight, then feel free to track it as a separate issue for a post-GA tech debt release.

Reply:
Tracked here: https://github.com/Azure/azure-functions-durable-python/issues/231